queries/meson/highlights.scm

```diff
@@ -0,0 +1,55 @@
+(comment) @comment
+(number) @number 
+(bool) @boolean
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+]
+@punctuation.bracket
+
+[
+  "="
+  "=="
+  "!="
+  "and"
+  "+"
+  "+="
+  "not"
```

Comment:
`@keyword.operator`

Reply:
Same for `"and"`

Reply:
> Same for `"and"`

What is same for and mean?

Reply:
friendly ping @@

Reply:
`@keyword.operator` for `"not"` and `"and"`. `@keyword` for all signs

Reply:
does it have `"or"`?

Reply:
> does it have `"or"`?

Thanks, it does have, I have forgotten it